### ISSUE TITLE
eval with specific threshold

### ISSUE THREAD

**Issue by AlexandreSmadja** [sovit-123/fasterrcnn-pytorch-training-pipeline#67]
Is there a way to get the precision/recall for a specific threshold ? Or a way to find the best threshold for my model

**Comment by sovit-123** [comment]
Yes, it is possible, but right now `eval.py` does not have an argument parser to control it from the command line.
You can use the `iou_thresholds` argument while initializing `MeanAveragePrecision` to set the threshold you want.

**Comment by supriamir** [comment]
hi @sovit-123 How do I get the inferences time for the test dataset?
